packages/manager/apps/web/client/app/domain/translations/Messages_fr_FR.json

```
@@ -819,8 +819,10 @@
   "domain_dashboard_options_status_disabled": "Résilié",
   "domain_dashboard_options_unavailable": "Option non disponible",
   "domain_dashboard_options_desactivate_option": "Résilier",
+  "domain_dashboard_options_dns_anycast_expiry": "date d'effet",
```

Where is this used ?

This looks like it should be 
```suggestion
  "domain_dashboard_options_dns_anycast_expiry": "Date d'effet",
```
But I can't be sure without context

---

This is how is it used: `Activé (date d'effet: 5 août 2020)`

---

Ok then could you please do this instead:
```suggestion
  "domain_dashboard_options_dns_anycast_expiry": "date d'effet: {{date}}",
```
This would give more context to translators :)

---

This is done. Thanks for the suggestion :)